More info about the ioc key (#12436)
demisto/content: Packs/ThreatGrid/ReleaseNotes/1_2_2.md

#### Integrations
##### Cisco Threat Grid

> - Fixed an issue where ***threat-grid-get-analysis-iocs*** was return an error in case there were no indicators, or it did not have a score

Reviewer: ```suggestion
- Fixed an issue where the ***threat-grid-get-analysis-iocs*** command returned an error in case there were no indicators, or indicators without a score.
```
